src/avio.jl

```diff
@@ -858,7 +858,7 @@ function seek_trim(r::VideoReader, seconds::Number)
     time_base = convert(Rational, stream.time_base)
     time_base == 0 && error("No time base")
     target_pts = seconds_to_timestamp(seconds, time_base)
-    frame_rate = convert(Rational, av_stream_get_r_frame_rate(stream))
+    frame_rate = convert(Rational, stream.r_frame_rate)
```

Comment:
Tests are failing so I can only assume that this is not the solution?

Reply:
This seems like the correct change based on https://patchwork.ffmpeg.org/project/ffmpeg/patch/20171023160821.6384-2-jamrial@gmail.com/

And the test failures could be rounding assumptions. Not looked deeper into it
```
Encoding with frame rate 29.5: Test Failed at /home/runner/work/VideoIO.jl/VideoIO.jl/test/writing.jl:196
  Expression: parse(Float64, measured_dur_str[1]) == target_dur
   Evaluated: 3.389831 == 3.39
```
```
Encoding with frame rate 29.5: Test Failed at /home/runner/work/VideoIO.jl/VideoIO.jl/test/writing.jl:214
  Expression: parse(Float64, measured_dur_str[1]) == target_dur
   Evaluated: 3.389831 == 3.39
```

Reply:
We're a bit far from rounding error? Maybe it's a difference of 1 frame?

Reply:
I think this is just correct
```
julia> 100 / 29.5
3.389830508474576

julia> 99 / 29.5
3.3559322033898304

julia> 101 / 29.5
3.4237288135593222
```

Perhaps `ffmpeg -v error -show_entries format=duration -of default=noprint_wrappers=1:nokey=1` just returns more precision on newer versions.

Reply:
I've pushed a change to the full precision